includes/subscriptions/class-wc-payments-subscriptions.php

```diff
@@ -157,4 +159,15 @@ public static function is_duplicate_site() {
 
 		return class_exists( 'WCS_Staging' ) && WCS_Staging::is_duplicate_site();
 	}
+
+	/**
+	 * Disable the WCPay Subscriptions feature on WooPayments plugin update if it's enabled and the store is no longer eligible.
+	 *
+	 * @see WC_Payments_Features::is_wcpay_subscriptions_eligible() for eligibility criteria.
+	 */
+	public static function maybe_disable_wcpay_subscriptions_on_update() {
+		if ( WC_Payments_Features::is_wcpay_subscriptions_enabled() && ! WC_Payments_Features::is_wcpay_subscriptions_eligible() ) {
```

Comment:
I wonder if we need to change this to also disable this feature if the Woo Subscriptions plugin is active.

Imagine there's a site that has used WCPay Subscriptions in the past, installed Woo Subscriptions, if they deactivate Woo Subscriptions do we want them to automatically switch to using WCPay Subscriptions feature?

Reply:
In line with the discussion here (p1695182789067919/1695179380.867469-slack-C055WHLA98D), Brent agrees and we'll disable the WCPay Subscriptions feature if its enabled and the Woo Subscriptions plugin is already active at the time of next WooPayments update.